app/javascript/packages/verify-flow/steps/password-confirm/password-confirm-step.tsx

```diff
@@ -98,7 +98,12 @@ function PasswordConfirmStep({ errors, registerField, onChange, value }: Passwor
           }),
           {
             button: ({ children }) => (
-              <HistoryLink basePath={stepPath} step={FORGOT_PASSWORD_PATH}>
+              <HistoryLink
+                basePath={stepPath}
+                step={FORGOT_PASSWORD_PATH}
+                aria-label={t('idv.forgot_password.link_html').replace(`%{link}`, '')}
```

Comment:
I'm trying to understand this change. Re-reading the ticket, it sounds like it was suggested that the "Forgot password?" text was not being read by the screen reader, but it seems like it is correct in my testing? It's not _focusable_ nor directly associated with the field, but it is reachable content.

![image](https://user-images.githubusercontent.com/1779930/185117582-60b8be01-7b73-4e09-bf09-37b666f09cd8.png)

My concern with this change then is that...

1. The text would be present and read twice: "Forgot password? Forgot password?"
2. This would create a disconnect between the visible and accessible name ([G211](https://www.w3.org/WAI/WCAG22/Techniques/general/G211)), making it difficult for users who use dictation software to interact with the link

I do think the link could be improved though, since its purpose isn't really clear by the link alone ([SC 2.4.9](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-link-only.html)), which may make it difficult to use for those who use a tool like web rotor to navigate links or who navigate by tabbing with a screen reader. A label like "Follow these instructions to reset your password" could help make it clearer.

Reply:
> Reading the ticket, it sounds like it was suggested that the "Forgot password?" text was not being read by the screen reader, but it seems like it is correct in my testing?
  
I can confirm that when the whole page is read (on `main`), the "Forgot password?" text is also being read. And I agree that the ticket could be more specific. My assumption was that the issue referred to the forgot password text being associated with the link that follows (when focused).
  
The same applies to the other part of the ticket, the social security example. It's reachable by the normal reading of the page but will not be read when either the input field or its label is selected/focused (again, on `main`).
  
> The text would be present and read twice: "Forgot password? Forgot password?"
Yes, this definitely happens on a full page read (but not when tabbing to the link). Perhaps there is another strategy we could use here.
  
> I do think the link could be improved though, since its purpose isn't really clear by the link alone ([SC 2.4.9](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-link-only.html)), which may make it difficult to use for those who use a tool like web rotor to navigate links or who navigate by tabbing with a screen reader. A label like "Follow these instructions to reset your password" or just "Reset your password" could help make it clearer.
  
It might be worth kicking the design back to UX real quick also, since as you've said the link is not ideal for accessibility purposes by its design.
  
_Side Note_: I just noticed that the "Follow these instructions" link cannot be focused with tab in Safari for some reason.

Reply:
> _Side Note_: I just noticed that the "Follow these instructions" link cannot be focused with tab in Safari for some reason.

Yeah this is unfortunately a default behavior in Safari, one which you can correct in Preferences > Advanced.

![Screen Shot 2022-08-17 at 1 09 31 PM](https://user-images.githubusercontent.com/1779930/185200819-65e68f38-d524-49e5-a0b5-f65f100497bc.png)